src/lamp_py/tableau/jobs/glides.py

```diff
@@ -0,0 +1,249 @@
+from typing import Optional
+from datetime import datetime
+from datetime import timezone
+
+import pyarrow
+import pyarrow.parquet as pq
+import pyarrow.dataset as pd
+from pyarrow.fs import S3FileSystem
+
+import polars as pl
+
+from lamp_py.tableau.hyper import HyperJob
+from lamp_py.runtime_utils.remote_files import (
+    glides_trips_updated,
+    glides_operator_signed_in,
+    tableau_glides_all_trips_updated,
+    tableau_glides_all_operator_signed_in,
+)
+from lamp_py.ingestion.glides import TripUpdates, OperatorSignIns
+from lamp_py.aws.s3 import file_list_from_s3, file_list_from_s3_with_details, object_exists
+
+
+# these are re-defined here for clarity. These will be asserted against
+glides_trips_updated_schema = pyarrow.schema(
```

Comment:
This schema doesn't match what you're outputting from `create_trips_updated_glides_parquet`